app/views/homepage/_government_activity.html.erb

```diff
@@ -39,12 +43,9 @@
       </div>
 
       <%# 
-        The "big number" component links below are hardcoded separately (i.e. not in a loop in this instance) and therefore their 'index' properties are also
-        hardcoded. Additionally, in order to account for these hardcoded components in the 'index_total' property, 2 is added to the ga4_links_counts variable 
-        (which is defined in homepage > index.html.erb). If a new "big number" component is added or removed, this will need to be reflected in the 'index' and
-        'index_total' properties to ensure that they remain accurate for tracking/analytics purposes.
-      %>
-      
+        The "big number" component links below are hardcoded separately (i.e. not in a loop in this instance) and therefore their 'index' properties are also hardcoded. If a new "big number" component is added or removed, this will need to be reflected in the 
+        'index_link' and 'index_total' properties to ensure that they remain accurate for tracking/analytics purposes.
```

Comment:
Quick question: seeing as the `index_section` and `index_section_count` properties are also hardcoded, do you think there would there be any mileage in putting a comment in to flag that they'll also need updating if a section is added/removed from the homepage? For example, wherever `index`/`index_values` is being used in the templates, we put something like `The hardcoded properties will need updating whenever sections are added/removed to ensure that tracking/analytics remains accurate.`

Also nitpick: the formatting of the comment is a bit odd

Reply:
Great shout, thanks @JamesCGDS. Have put a comment in the index template, and updated the formatting of this comment as well.